### ISSUE TITLE
Время жизни учетных данных(анкеты)

### ISSUE THREAD

**Issue by tebaly** [freedomsex/freedomsex#497]
Так сложилось, что анкетные данные не обязательно связаны с учетными данными. Иначе говоря, при невозможности войти на сайт по любым причинам, в том числе связанным с удалением учетных данных, анкета ещё некоторое время будет _жить_. Но большинство Пользователей не в состоянии переварить сие, и считают анкетой всё что есть на сайте под их ID...

Данным постом затрагивается именно время жизни учетных данных.
* Теперь оно становится ограниченным в 30 дней (затем до 15 или 7 дней)
* Если не указан/подтвержден емайл [в будущем +соцсети] - доступ может быть утерян

Таким образом, _указывать емайл не обязательно_, но доступ к анкете становится ограничен по времени и будет утерян по его истечению **безвозвратно**. Если пользователь укажет емайл и подтвердит его, то будет возможность восстановить доступ через напоминание пароля.

_Вероятно доступ будет оставаться до самостоятельного разлогивания Пользователя. Но это не точно..._

**Comment by inquisitor6789** [comment]
а чего мелочимся? Давайте уже сканы паспортов собирать :-)

**Comment by tebaly** [comment]
@inquisitor6789 речь о OAuth 2.0 - войти с помощью. Это для тех, кому удобнее. Если вам удобнее по паспорту, то этот вопрос можно обдумать. Сложнее, дольше, но не так уже и фантастично :stuck_out_tongue:

**Comment by inquisitor6789** [comment]
давайте пойдем по пути эпла - отпечатки пальцев, радужка глаза, морда лица :-) еще надо анализ мочи и крови 😈

**Comment by tebaly** [comment]
За всё время существования проекта к Администраторам не выдвигалось ни одного требования. Будь-то верификация номера телефона, отправка сканов паспорта, личная встреча и тому подобное. Вы будете смеяться [нет], но все эти этапы заложены в Систему безопасности ещё лет семь назад (со дня её основания). Были попытки мягонько пощупать реакцию Администраторов на данную стратегию. И здесь "или/или", так как некоторые уровни приватности не позволяют иного в принципе. Благо проект развивается так медленно, что до того может и не дойти...

Верификация же Пользователей нужна самим Пользователям, как это не раз упоминал многоуважаемый @mj4444ru. Обязательности здесь вероятно не будет, но иначе порядок навести может и не получиться. Пробовать будем, безусловно. Мочу пересылать затратно, но вот иные вариации вполне ебабеленые.

**Comment by Grian59** [comment]
Да, время жизни анкеты нужно ограничить даже не 7 днями, а 3-мя. 
у меня 85% корреспондентов одноразовые. Зачем вам хранить их? Тем более при такой легкой регистрации
